### PR TITLE
mpc85xx: split up DEVICE_TITLE

### PR DESCRIPTION
**Pull request by mwarning**
Splits up DEVICE_TITLE into DEVICE_VENDOR, DEVICE_MODEL and DEVICE_VARIANT.

### CODE REVIEW

File: target/linux/mpc85xx/image/Makefile [hunk 1]
```diff
@@ -67,7 +68,9 @@ endef
 TARGET_DEVICES += tl-wdr4900-v1
 
 define Device/red-15w-rev1
-  DEVICE_TITLE := Sophos RED 15w Rev.1
+  DEVICE_VENDOR := Sophos
+  DEVICE_MODEL := RED 15w
+  DEVICE_VARIANT := Rev.1
```
Comment (mwarning) [inline target/linux/mpc85xx/image/Makefile]: v1? I do not find information of other revisions..

File: target/linux/mpc85xx/image/Makefile [hunk 2]
```diff
@@ -47,7 +47,8 @@ endef
 ifeq ($(SUBTARGET),generic)
 
 define Device/tl-wdr4900-v1
-  DEVICE_TITLE := TP-Link TL-WDR4900
+  DEVICE_VENDOR := TP-Link
+  DEVICE_MODEL := TL-WDR4900
```
Comment (CHKDSK88) [inline target/linux/mpc85xx/image/Makefile]: I suggest add ``DEVICE_VARIANT := v1``. Here You can find v2:
https://github.com/openwrt/openwrt/blob/dcb5cf85c8196b2ba35a0aa395f2247ed2163121/target/linux/ath79/image/generic-tp-link.mk#L249

File: target/linux/mpc85xx/image/Makefile (hunk 1 above)
Comment (blocktrron) [reply target/linux/mpc85xx/image/Makefile]: There's only one (to my knowledge). However, the Rev. 1 is clearly labled on the device, so i would keep it.

Comment (adrianschmutzler) [reply target/linux/mpc85xx/image/Makefile]: One could think about adding a space between Rev. and 1, but this is really a nitpick.

**Review by adschm — APPROVED** [review]
LGTM now.

File: target/linux/mpc85xx/image/Makefile (hunk 1 above)
Comment (blocktrron) [reply target/linux/mpc85xx/image/Makefile]: It's indeed how SOPHOS has written it on the device.

Comment (mwarning) [reply target/linux/mpc85xx/image/Makefile]: @blocktrron "Rev.1" or "Rev. 1"?

Comment (blocktrron) [reply target/linux/mpc85xx/image/Makefile]: "Rev.1"

**Comment by blocktrron** [comment]
Thanks! Pulled into my staging tree at https://git.openwrt.org/openwrt/staging/blocktrron.git

**Pull request closed without merging**